examples/light-switch-app/nrfconnect/README.md

```diff
@@ -273,6 +298,11 @@ platform image.
     with 1% increments every 300 milliseconds as long as **Button 2** is
     pressed.
 
+**Button 3** can be used for the following purposes:
+
+-   _Pressed once_ &mdash; Changes the value of the attribute CurrentPosition
```

Comment:
use backticks to escape things that are not words

Reply:
Done.